core/src/avm2/globals/flash/utils.as

```diff
@@ -8,14 +8,7 @@ package flash.utils {
 	// made specifically to fool com.adobe.serialization.json.JsonEncoder.
```

Comment:
The comment block here is no longer applicable

Reply:
It's mostly still applicable, but should be moved to the .rs file.